core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpMeasurementScheduler.java

```diff
@@ -564,7 +564,7 @@ void createMeasurementEvent(
                         case LineTcpParser.ENTITY_TYPE_INTEGER:
                             Unsafe.getUnsafe().putByte(bufPos, entity.getType());
                             bufPos += Byte.BYTES;
-                            Unsafe.getUnsafe().putLong(bufPos, entity.getIntegerValue());
+                            Unsafe.getUnsafe().putLong(bufPos, entity.getLongValue());
```

Comment:
Can you please explain why it should be `getLongValue` here?

Reply:
Because return value was and still is `long` and not Integer. We also had separate field `timestamp` which was used as long and prohibited code reuse